examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
```

Comment:
@davidfurlong @stephancill the issue here is that next.js fails to compile the route because it somehow doesn't allow to use react in route handlers. Maybe there is a way to work around this but I couldn't find one quickly.